### DISCUSSION TITLE
ERIGON HIGH IO ISSUES

### DISCUSSION THREAD

**Discussion by ggangisetti** [erigontech/erigon#2758] [Q&A] [answered]
Hi,

I am having a high IO issue for erigon full nodes. I am running the rpc daemon and Using NVmes as storage.

There has been a suggestion to run the container without `--datadir` which seems to have worked for other node operators.

Please help me with the issue.

Thanks

**Comment by AskAlexSharov** [comment]
@ggangisetti 
- erigon version?
- os version?
- FS version?
- "There has been a suggestion to run the container without --datadir" - it was for some specific version of erigon, not on latest versions.
- what means "high"? "higher than before?" or "we expected less?"

**Comment by ggangisetti** [comment, marked as answer]
I upgraded them to 2021.09.04 and working fine

Right now my erigon version I am upgrading it to latest 2021.10.02
Os -- MACOS BIG SUR - 11.5.1
Dont know the FS version
High means previously there is no issue. higher than before
but right now I see no issue may be upgrading to new version helped I guess
